# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/epsilon/png-compile-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

Yeah - the usual libs missing ...

  - Rene Rebe <rene@rocklinux.org>

--- epsilon-2004-01-18/configure.in.orig	2004-02-26 16:12:34.000000000 +0100
+++ epsilon-2004-01-18/configure.in	2004-02-26 16:12:51.000000000 +0100
@@ -66,7 +66,7 @@
 AC_CHECK_HEADER( png.h, 
   [ AC_DEFINE(HAVE_PNG_H, 1, [png.h available]) 
     have_png="yes" 
-    LIBS="$LIBS -lpng" ], 
+    LIBS="$LIBS -lpng -lz" ], 
   [ AC_MSG_ERROR(Cannot find png.h on your system, try --with-png=) ])
 AC_CHECK_LIB(png, png_create_read_struct, [ 
     AC_DEFINE(HAVE_PNG_LIB, 1, [libpng available]) 
